### DISCUSSION TITLE
How to change Win11's Start Menu blur?

### DISCUSSION THREAD

**Discussion by mkanet** [MicaForEveryone/MicaForEveryone#371] [Q&A] [answered]
I am successfully able to add "Tabbed" backdrop to various apps.  However, I can't figure out how to change the default "Mica" blur on Win11 Start Menu to "Tabbed" or "Acryllic".  I tried specifying the class:  `Windows.UI.Core.CoreWindow` in Mica for Windows.  However, that didnt make any difference.

If it is not possible to do this with MicaForEveryone, is there another utility that can do this without replacing the Win11 Start Menu completely?

**Comment by dongle-the-gadget** [comment]
MFE doesn't allow changing that, but I've heard that TranslucentSM can.

**Reply by mkanet** [reply]
Thanks for the quick reply.  I've never heard of TranslucentSM, can you point it to me?  Maybe you're talking about TranslucenTB?  I still use TranslucentTB for my Win11 taskbar; however it can't modify the transparency for the Win11 Start Menu; only WIn10... at least for me.

**Reply by dongle-the-gadget** [reply, marked as answer]
https://github.com/rounk-ctrl/TranslucentSM
